Apps/Sandcastle/gallery/CZML Polygon - Intervals, Availability.html

```diff
@@ -0,0 +1,159 @@
+<!DOCTYPE html>
```

Comment:
What is the intention with this example?  There is a lot of flicker:
![ok](https://cloud.githubusercontent.com/assets/782098/10169479/3e9f3be0-669f-11e5-82a9-381a875a6e79.gif)